sensor_msgs/PointCloud2[] clusters
geometry_msgs/PoseArray clusterCentroids
geometry_msgs/Point sensorPosition
---
ClassificationResult[] classificationResults